OpenBazaar/openbazaar-desktop
# Migrate pre mw

This change-set handles migrating pre-multiwallet built-in configurations. It will use the walletCurrency to determine which data directory to pass to the server.

To test:
- delete you data dirs
- clear your local storage
- roll back the server to the latest release, commit `0f64313` and `go build` an executable
- in main.js fudge `serverPath` and `serverFilename` to point to the old server executable
- start up `master` on the client
- create a config for BTC and one for BCH. If you're feeling extra froggy, create a ZEC one as well
- checkout the latest master on the server and build an executable from it. Ensure `main.js` references that executable.
- start up the client on this PR
- the servers you created perviously should all work, albeit now when you go to the Wallet, they should be multi-wallets.
